cloudflare.loader.php

```diff
@@ -10,14 +10,18 @@
 }
 
 // Rewrites Cloudflare IP
-$ipRewrite = new IpRewrite();
-
-$isCf = $ipRewrite->isCloudFlare();
-if ($isCf) {
-    // Fixes Flexible SSL
-    if (isset($_SERVER['HTTP_X_FORWARDED_PROTO']) && $_SERVER['HTTP_X_FORWARDED_PROTO'] == 'https') {
-        $_SERVER['HTTPS'] = 'on';
+try {
+    $ipRewrite = new IpRewrite();
+
+    $isCf = $ipRewrite->isCloudFlare();
+    if ($isCf) {
+        // Fixes Flexible SSL
+        if (isset($_SERVER['HTTP_X_FORWARDED_PROTO']) && $_SERVER['HTTP_X_FORWARDED_PROTO'] == 'https') {
+            $_SERVER['HTTPS'] = 'on';
+        }
```

Comment:
Only `IpRewrite` function throws exception. I included the whole block under try catch to make it simpler to understand it's a block.